CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.0.0-alpha.0
+
+* `FEAT`: rewrite project to new, [Preact](https://github.com/preactjs/preact)-based architecture

```

Comment:
Add fundamental rework of UI.
Add removal of tabs.
Add dirty markers being introduced.